src/main/groovy/edu/oregonstate/mist/api/Resource.groovy

```diff
@@ -136,16 +136,18 @@ abstract class Resource {
      * The path is copied from the request path.
      * The query parameters are taken from the params argument.
      *
-     * Falsey params are omitted from the url.
+     * Falsely params are omitted from the url.
```

Comment:
Ah yes. I've seen it as falsy too. I guess I should read a whole sentence before correcting a word.